### ISSUE TITLE
Create POSTMAN feature tests for server routes.

### ISSUE THREAD

**Issue by urosjarc** [Danilo-Danijel-team/sms_sender#12]
- [ ] After #11 you can create postman feature tests that will test if the server correctly implements all the features required by the application.
- [ ] First make sure that all tests in #11 passes, if not create new issue for @danilojezernik to resolve.
- [ ] Create tests that will be as independent as humanly possible.
- [ ] Make feature tests together with @danilojezernik.
- [ ] If you find something problematic or bug, create new issue for @danilojezernik to resolve
